.github/workflows/build.yml

```diff
@@ -5,9 +5,9 @@ name: (Blocking) Test without updating any packages
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, pcp-zeroconf ]
```

Comment:
I think we can remove the main branch here?

Reply:
Still at the end, we will have all the changes to the main branch, and it would be overhead to add `main` again while merging this branch to the main. I think it is good to not remove it completely.